### PR TITLE
[WIP] Org permissions tab tests

### PR DESCRIPTION
**Pull request by dsesami**
##### SUMMARY
Tests for new views for organization-level permissions in teams tab/ orgs tabs

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME

 - UI

##### AWX VERSION

```
AWX 2.0.1
```

### CODE REVIEW

**Comment by dsesami** [comment]
admin permissions are broken (see https://github.com/ansible/awx/issues/2563) so this test will test both admin and non-admin permissions upon resolution of that issue.

**Comment by dsesami** [comment]
This is currently blocked by https://github.com/ansible/awx/issues/2601 -- once this is consistent, final steps can be added and this can be merged.

File: awx/ui/test/e2e/fixtures.js [hunk 1]
```diff
@@ -299,7 +299,19 @@ const getAuditor = (namespace = session) => getOrganization(namespace)
 
 const getUser = (namespace = session) => getOrganization(namespace)
     .then(organization => getOrCreate(`/organizations/${organization.id}/users/`, {
-        username: `user-${uuid().substr(0, 8)}`,
+        username: `${namespace}-user-${uuid().substr(0, 8)}`,
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/fixtures.js:302]: we need to keep the `namespace` out of the username, as some tests (e.g: xss) will fail due to server-side validation of usernames.

**Review by jakemcdermott — CHANGES_REQUESTED** [review]

File: awx/ui/test/e2e/fixtures.js (hunk 1 above)
Comment (dsesami) [reply awx/ui/test/e2e/fixtures.js:302]: I think there might be validation issues with that breaking other older tests(?) but I can investigate its effects first thing tomorrow.

**Comment by dsesami** [comment]
closing this PR and opening a new one because I'm not sure what commits went into this, i think it's a branch mixup.

**Pull request closed without merging**